pkg/lib/cockpit.js

```diff
@@ -2699,6 +2714,25 @@ function factory() {
         visibility_change();
     }());
 
+    /* ---------------------------------------------------------------------
+     * Dark mode
+     */
+    (function() {
+        if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+            cockpit.dark_mode = true;
+        } else {
+            cockpit.dark_mode = false;
+        }
+
+        window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', event => {
+            if (event.matches) {
+                cockpit.dark_mode = true;
+            } else {
+                cockpit.dark_mode = false;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17319-20221028-143606-093d853b-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)